src/main/java/com/iota/iri/LedgerValidator.java

```diff
@@ -265,11 +265,7 @@ public boolean updateDiff(Set<Hash> approvedHashes, final Map<Hash, Long> diff,
         if (currentState == null) return false;
         boolean isConsistent = Snapshot.isConsistent(milestone.latestSnapshot.patchedDiff(currentState));
         if (isConsistent) {
-            currentState.forEach((key, value) -> {
-                if(diff.computeIfPresent(key, ((hash, aLong) -> value + aLong)) == null) {
-                    diff.putIfAbsent(key, value);
-                }
-            });
+            currentState.forEach((key, value) -> diff.compute(key, ((hash, aLong) -> hash == null ? value: aLong + value)));
```

Comment:
Style:
`value:` -> `value :`

I also personally think that 2 lines are more readable than 1 long line